web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx

```diff
@@ -235,12 +244,25 @@ const FieldUpdateModal: React.FC<Props> = ({
             )}
             <Form.Item
               name="multiValue"
+              valuePropName="checked"
               extra={t("Stores a list of values instead of a single value")}>
               <Checkbox>{t("Support multiple values")}</Checkbox>
             </Form.Item>
           </TabPane>
           <TabPane tab={t("Validation")} key="validation" forceRender>
             <FieldValidationInputs selectedType={selectedType} />
+            <Form.Item
+              name="required"
+              valuePropName="checked"
+              extra={t("Prevents saving an entry if this field is empty")}>
```

Comment:
Prevent saving an entry if this field is empty

Reply:
@KaWaite, updated.

Reply:
Sorry my bad. I am mixing writing styles. Can you revert this? The description for Unique is `Ensures that....`, so here should be `Prevents saving...`.
My bad. Apologies

Reply:
@KaWaite , updated.